docs/ota-client-guide/modules/ROOT/pages/meta-updater-build.adoc

```diff
@@ -0,0 +1,62 @@
+= Build
+:meta-updater-github-url: https://github.com/advancedtelematic/meta-updater/tree/master
+
+== Quickstart
+
+If you don't already have a Yocto project that you want to add OTA to, you can use the xref:dev@getstarted::raspberry-pi.adoc[HERE OTA Connect Quickstart] project to rapidly get up and running on a Raspberry Pi. It takes a standard https://www.yoctoproject.org/tools-resources/projects/poky[poky] distribution, and adds OTA and OSTree capabilities.
+
+== Dependencies
+
+//MC: TOMERGE: These "dependencies" mostly just duplicates the prerequisite sections: https://main.gitlab.in.here.com/olp/edge/ota/documentation/ota-connect-docs/blob/master/docs/getstarted/modules/ROOT/pages/raspberry-pi.adoc
+
+In addition to the link:https://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#required-packages-for-the-build-host[standard Yocto dependencies], meta-updater generally requires a few additional dependencies, depending on your use case and target platform. To install these additional packages on Debian/Ubuntu, run this:
+
+....
+sudo apt install cpu-checker default-jre parted
+....
+
+To build for https://github.com/advancedtelematic/meta-updater-minnowboard[Minnowboard] with GRUB, you will also need to install https://github.com/tianocore/tianocore.github.io/wiki/OVMF[TianoCore's ovmf] package on your host system. On Debian/Ubuntu, you can do so with this command:
+
+....
+sudo apt install ovmf
+....
```

Comment:
It looks like this whole doc can probably be dropped as long as any unique information is preserved elsewhere. I think the first two sections ("Quickstart" and "Dependencies") are probably the most important parts that should be left in the meta-updater README. I'll make that change on the meta-updater PR.

Reply:
Ok, let's say we keep it in there for now and prune it in a second round (just tryna make sure the incoming meta-updater README links work with this PR)